src/main/java/com/uber/cadence/client/WorkflowOptions.java

```diff
@@ -192,6 +196,12 @@ public Builder setMemo(Map<String, Object> memo) {
       return this;
     }
 
+    /** Specifies additional indexed information in result of list workflow. */
```

Comment:
Which types are allowed as values? Are they serialized in any way?

Reply:
The same question applies to memo.

Reply:
Comments updated